src/api/notes/index.ts

```diff
@@ -2,7 +2,6 @@ import { defaultFetchConfig, expectResponseCode, getApiUrl } from '../utils'
 
 interface LastChange {
```

Comment:
I'm still quite unhappy about the whole Notes-interface. In my opinion it's too directly copied from 1.6 and doesn't respect newer things like the permissions of a note etc.

I just didn't want to change that in this PR, because I guess it'll be a more relevant problem/topic of the backend integration.